photoeditor/build.gradle

```diff
@@ -2,14 +2,12 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-
+    namespace 'ja.burhanrashid52.photoeditor'
     compileSdkVersion 31
 
     defaultConfig {
         minSdkVersion 14
         targetSdkVersion 31
-        versionCode 7
```

Comment:
Change made by Upgrade Assistant

> Remove ignored configuration properties
>
> The build configuration used to allow setting of properties for some modules, even where those properties do not have any effect for that kind of module. In some circumstances from now, attempts to set those properties may cause errors rather than being silently ignored.